overlays/imx8qm.nix

```diff
@@ -2,4 +2,30 @@
 final: prev: {
   linux_imx8 = final.callPackage ./bsp/kernel/linux-imx8 { pkgs = final; };
   inherit ( final.callPackage ./bsp/u-boot/imx8qm/imx-uboot.nix { pkgs = final; }) ubootImx8 imx-firmware;
+
+  spectrum-live = prev.spectrum-live.overrideAttrs (old: {
+    KERNEL = final.linux_imx8;
+    pname = "build/live.img";
+    installPhase = ''
+      runHook preInstall
+      dd if=/dev/zero bs=1M count=6 >> $pname
+      partnum=$(sfdisk --json $pname | grep "node" | wc -l)
+      while [ $partnum -gt 0 ]; do
+        echo '+6M,' | sfdisk --move-data $pname -N $partnum
```

Comment:
the magic `+6M` warrants a comment line - like line 20

Reply:
Addressed